### PR TITLE
feat: added different types of snuba consumers

### PR DESCRIPTION
**Pull request by Mokto**
I'm still a bit unsure about the necessity of this

EDIT: seems like it consumed a lot of messages when it started, so it did something.

### CODE REVIEW

File: sentry/templates/deployment-snuba-outcomes-consumers.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
```
Comment (J0sh0nat0r) [inline sentry/templates/deployment-snuba-outcomes-consumers.yaml]: This file should probably be named `snuba-outcomes-consumer` to fit with the others

File: sentry/templates/deployment-snuba-outcomes-consumers.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        volumeMounts:
+        - mountPath: /etc/snuba
+          name: config
+          readOnly: true
+        resources:
+{{ toYaml .Values.snuba.outcomesConsumer.resources | indent 12 }}
+      volumes:
+        - name: config
+          configMap:
+            name: {{ template "sentry.fullname" . }}-snuba
```
Comment (J0sh0nat0r) [inline sentry/templates/deployment-snuba-outcomes-consumers.yaml]: Should probably end with a new line here

**Review by J0sh0nat0r — COMMENTED** [review]
Other than the aforementioned LGTM 👍

File: sentry/templates/deployment-snuba-outcomes-consumers.yaml (hunk 1 above)
Comment (Mokto) [reply sentry/templates/deployment-snuba-outcomes-consumers.yaml]: 👍

**Pull request merged**